### DISCUSSION TITLE
Additional Information Field

### DISCUSSION THREAD

**Discussion by joseph-kendall** [CMSgov/price-transparency-guide#407] [Q&A] [answered]
@shaselton-usds 

For contracts that have lesser of language, stop loss, days based on a range, a range based on line charges, and extra information about a transplant, can we add these in the Additional Information field to give more color to the contract?

**Comment by shaselton-usds** [comment, marked as answer]
@joseph-kendall At this point, yes.

I would point to the notes in the PR: https://github.com/CMSgov/price-transparency-guide/pull/393#issue-1150749583. We are looking to add this type of support to allow for the documentation that you're talking about here. Would there be any other attributes needed to support your scenarios?
